### PR TITLE
Game Refactor & README Update

### PR DESCRIPTION
**Pull request by whitneykidd**
- Take methods from Game class and move them into Game_Stats class, along with their tests. 
- Updated Stat_Tracker tests to reflect these changes
- Updated README with project organization overview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
-# Futbol
-
-Starter repository for the [Turing School](https://turing.io/) Futbol project.
+# Fútbol
```
Comment (dborski) [inline README.md:1]: Accents in markdown? I think this is going to get us a 4.

File: lib/game_stats.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+require_relative 'mathable'
+
+class GameStats
```
Comment (dborski) [inline lib/game_stats.rb:3]: This GameStats class does make our project more consistent and organized.

**Review by dborski — APPROVED** [review]

File: lib/game_stats.rb (hunk 2 above)
Comment (IamNorma) [inline lib/game_stats.rb:3]: Perfect. Now all stats classes are consistent.

File: lib/team_stats.rb [hunk 3]
```diff
@@ -10,6 +10,13 @@ def initialize(game_teams, teams, games)
     @games = games
   end
 
+  def find_name(id)
+    team_name = @teams.find do |team|
+      team.team_id == id
+    end
+    team_name.team_name
+  end
+
```
Comment (kiverso) [inline lib/team_stats.rb:19]: This feels much cleaner than the class method previously used.

File: test/league_test.rb [hunk 4]
```diff
@@ -1,24 +0,0 @@
...
-
-  def test_it_exists
-    assert_instance_of League, @league
-  end
-
-  def test_it_has_readable_attributes
-    assert_equal './data/teams.csv', @league.teams_path
-    assert_equal './data/game_teams.csv', @league.game_teams_path
-  end
-end
```
Comment (kiverso) [inline test/league_test.rb:24]: Nice catch, I had almost forgotten this abandoned class. Glad we caught this and deleted it

**Pull request merged**